device/samples/package.json

```diff
@@ -1,22 +1,22 @@
 {
-  "name": "iot-device-samples",
-  "version": "0.0.1",
-  "private": true,
-  "description": "Simple samples demonstrating how to use the Azure IoT device SDK for Node.js",
-  "author": "Microsoft Corp.",
-  "license": "MIT",
-  "dependencies": {
-    "azure-iot-device": "1.11.2",
-    "azure-iot-device-amqp": "1.10.3",
-    "azure-iot-device-http": "1.10.3",
-    "azure-iot-device-mqtt": "1.10.3",
-    "es5-shim": "^4.5.12",
-    "lodash": "^4.17.15"
-  },
-  "devDependencies": {
-    "jshint": "^2.10.2"
-  },
-  "scripts": {
-    "lint": "jshint --show-non-errors ."
-  }
-}
+    "name": "iot-device-samples",
+    "version": "0.0.1",
+    "private": true,
+    "description": "Simple samples demonstrating how to use the Azure IoT device SDK for Node.js",
+    "author": "Microsoft Corp.",
+    "license": "MIT",
+    "dependencies": {
+        "@azure/event-hubs": "^2.1.1",
+        "azure-iot-device": "^1.11.2",
+        "azure-iot-device-http": "^1.10.3",
+        "azure-iot-device-mqtt": "^1.10.2",
```

Comment:
we've gotta remove these carats from all the `azure-iot-*`. Because we have 'tight dependencies' on our published packages (i.e. we might break dependency between packages if they're not upgraded in lockstep), letting the minor or patch version float doesn't work.